portfolio/src/main/webapp/index.html

```diff
@@ -15,84 +15,99 @@
-      <img src="/images/canmore.jpg" alt="Canmore" style="width:100%">
-    </div>
-      <footer class="w3-container w3-blue">
-      <h5>I lived here from I was two and after I was 5. (2000-2002, 2006-2020)<h5>
-      </footer>
-    </div>
-    </div>
-      <div class="column">
-      <div class="w3-card-4" style="width:100%;">
-      <header class="w3-container w3-blue">
-      <h4>Kamchatka, Russia</h4>
-      </header>
-      <div class="w3-container">
-      <img src="/images/russia.jpg" alt="Russia" style="width:100%">
-    </div>
-      <footer class="w3-container w3-blue">
-      <h5>I lived here from the age of 2 to 5. (2002-2006)<h5>
-      </footer>
-    </div>
-    </div>
-      <div class="column">
-      <div class="w3-card-4" style="width:100%;">
-      <header class="w3-container w3-blue">
-      <h4>Kingston, Ontario</h4>
-      </header>
-      <div class="w3-container">
-      <img src="/images/kingston.jpeg" alt="Kingston" style="width:100%">
-    </div>
-      <footer class="w3-container w3-blue">
-      <h5>Since 2018 I have lived here while I attend university.(2018-2020)<h5>
-      </footer>
-    </div>
-    </div>
+      <div class="row">  
+        <div class="column">
+          <div class="w3-card-4" style="width:100%;">
+            <header class="w3-container w3-blue">                   
+              <h4>Canmore, Alberta<h4> 
+            </header>
+            <div class="w3-container">
+              <img src="/images/canmore.jpg" alt="Canmore" style="width:100%">
+            </div>
+            <footer class="w3-container w3-blue">
+              <h5>I lived here from I was two and after I was 5. (2000-2002, 2006-2020)<h5>
+            </footer>
+          </div>
+        </div>
+        <div class="column">
+          <div class="w3-card-4" style="width:100%;">
+            <header class="w3-container w3-blue">
+              <h4>Kamchatka, Russia</h4> 
+            </header>
+            <div class="w3-container">
+              <img src="/images/russia.jpg" alt="Russia" style="width:100%">
+            </div>
+            <footer class="w3-container w3-blue">
+              <h5>I lived here from the age of 2 to 5. (2002-2006)<h5>
+            </footer>
+          </div>
+        </div>
+        <div class="column">
+          <div class="w3-card-4" style="width:100%;">
+            <header class="w3-container w3-blue">
+              <h4>Kingston, Ontario</h4>
+            </header>
+            <div class="w3-container">
+              <img src="/images/kingston.jpeg" alt="Kingston" style="width:100%">
+            </div>
+            <footer class="w3-container w3-blue">
+              <h5>Since 2018 I have lived here while I attend university.(2018-2020)<h5>
+            </footer>
+          </div>
+        </div>
+      </div>
       &nbsp;&nbsp; 
-    </div>
+      <p> Guess what my favourite animal is <p>
+      <button onclick="displayImage()">Answer</button>
+      <div id="myDIV">
```

Comment:
Could we rename this to be more meaningful? Also prefer hyphen as a delimiter if needed, instead of capitalization: 
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Thank you for the feedback I just have a couple questions.  Firstly, as for the naming of the function I was wondering how I can change it to be more meaningful? From my perspective it makes sense why it would be named that. Could you provide me with an example of what would be a more meaningful name? Secondly, I noticed that you referenced the HTML style guide but since the function is a JavaScript function should I not be following style guide for that? From my understanding of naming functions in JavaScript you are supped to use capitalization. https://google.github.io/styleguide/jsguide.html#naming

Reply:
Perhaps github highlighting is causing issues, I was referring to the id "myDIV". Does that help more?

Reply:
That makes much more sense.  Thank you for the clarification!